ui/src/app/settings/components/certs-list/certs-list.scss

```diff
@@ -0,0 +1,45 @@
+@import 'node_modules/argo-ui/src/app/shared/styles/config';
+
+.certs-list {
+    &__top-panel {
+        padding: 1em;
+
+        & > .columns:first-child {
+            font-size: 8em;
+            text-align: center;
+        }
+
+        & > .columns:last-child {
+            text-align: center;
+            border-left: 2px solid $argo-color-gray-4;
+
+            & > p {
+                margin-bottom: 0;
+                margin-top: 24px;
+                &:first-of-type {
+                    font-size: 1.25em;
+                }
+            }
+
+            & > button {
+                width: 15em;
+            }
+        }
+    }
+
+    .argo-table-list {
+        .argo-dropdown {
+            float: right;
+        }
+    }
+
+
+}
+
+textarea.argo-field {
```

Comment:
Please move it inside `.certs-list` to make sure that style is applied only on `certs-list` page. Styles are loaded when user opens the page by stays loaded user reloads browser page.

Reply:
Done, thanks for the hint.